src/components/ble/NimbleController.cpp

```diff
@@ -45,16 +48,18 @@ NimbleController::NimbleController(Pinetime::System::SystemTask& systemTask,
 }
 
 void nimble_on_reset(int reason) {
-  NRF_LOG_INFO("Resetting state; reason=%d\n", reason);
+  NRF_LOG_INFO("Nimble lost sync, resetting state; reason=%d", reason);
 }
 
 void nimble_on_sync(void) {
-    int rc;
+  int rc;
 
-    rc = ble_hs_util_ensure_addr(0);
-    ASSERT(rc == 0);
+  NRF_LOG_INFO("Nimble is synced");
+
+  rc = ble_hs_util_ensure_addr(0);
```

Comment:
I think you could initialize and define on the same line here as well.

Reply:
I consider that bad style and we don't have a rule for it here